state/factory/workingset.go

```diff
@@ -30,33 +24,26 @@ type (
 	// WorkingSet defines an interface for working set of states changes
 	WorkingSet interface {
 		// states and actions
-		LoadOrCreateAccountState(string, *big.Int) (*state.Account, error)
-		Nonce(string) (uint64, error) // Note that Nonce starts with 1.
-		CachedAccountState(string) (*state.Account, error)
```

Comment:
i recall CachedState() is needed by evm/stateDB, how is that handled now you delete the CachedState()?